cuda_bindings/cuda/__init__.py

```diff
@@ -1,14 +0,0 @@
-def __getattr__(name):
-    if name == "__version__":
-        import warnings
-
-        warnings.warn(
-            "accessing cuda.__version__ is deprecated, " "please switch to use cuda.bindings.__version__ instead",
-            DeprecationWarning,
```

Comment:
@vzhurba01 I did a few quick searches on my cell and noticed that some projects still use this attribute. Maybe for consistency we should just change this to `UserWarning` instead of removing it entirely in 12.9.0? We can remove it in say 12.9.1 instead. WDYT?

Reply:
Introducing a breaking change like this is unideal in general in a minor / patch release. Can we defer removing it until the next major release?

Reply:
The major problem is we want `cuda` to be a namespace package, but the existence of `cuda/__init__.py` breaks a bunch of tools. We need to get rid of it sooner than later to not block any of our projects. Let me see if a `.pth` based hack can serve the same purpose.

Reply:
> Let me see if a .pth based hack can serve the same purpose.

Commit [2537375](https://github.com/NVIDIA/cuda-python/pull/493/commits/25373752904f4fd8c0d64c8713dad773b970f356) seems to do the trick!

> we should use FutureWarning instead of a generic UserWarning

Done in commit [cf28b44](https://github.com/NVIDIA/cuda-python/pull/493/commits/cf28b44d4a488012d8b2cd8dcba840e160c3f8fd)